src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -66,6 +101,8 @@
       <ExtraSourcesNuGetSourceName>ExtraSources</ExtraSourcesNuGetSourceName>
       <SourceBuildSources>$(SourceBuiltNuGetSourceName)</SourceBuildSources>
       <SourceBuildSources Condition="'$(ExtraRestoreSourcePath)' != ''">$(SourceBuildSources);$(ExtraSourcesNuGetSourceName)</SourceBuildSources>
+      <BuildCommand>$(BuildCommand) /p:RestoreConfigFile=@(NuGetConfigFiles)</BuildCommand>
```

Comment:
I think this should go into repo-projects/Directory.Build.props file after you removed the NuGetConfigFile item in favor of just a property. Same for the `ModifiedNugetConfigFilesIntermediateRoot` property then that you will need to construct the path.

Reply:
Yes, with removal of the item I am moving this to the props file. Will update the PR after I finish testing.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/55d2441f88af313329b757886fc4d72cf7c177c8